mlrun/api/migrations_mysql/versions/c905d15bd91d_notifications.py

```diff
@@ -49,17 +49,17 @@ def upgrade():
             "condition", sa.String(length=255, collation="utf8_bin"), nullable=False
         ),
         sa.Column("params", sa.JSON(), nullable=True),
-        sa.Column("run", sa.Integer(), nullable=True),
+        sa.Column("parent_id", sa.Integer(), nullable=True),
```

Comment:
worth leaving a comment why you decided to use parent id and not run id as this is a bit counter-intuitive, without understanding the reason (which is, afaik - due to implementation abstraction)

Reply:
Well, the idea is that this table is generated through a `make_notification` function that can be called for any DB object, so the table will be the same across objects (with a different name). This is a pattern we use in several other cases (tags, labels). I will add some comments here, though this is just the migration code, so usually won't be used for real reference - the actual DB schema code is clearer in that sense.